app/views/any_login/_any_login.html.erb

```diff
@@ -1,7 +1,21 @@
+<% case AnyLogin.asset_source
+   when :sprockets %>
+  <%= javascript_include_tag "any_login/application.js", async: true, data: {'turbo-track': 'reload'} %>
```

Comment:
did you consider just add

```
 get '/any_login/js' => 'application#js', as: :js
 get '/any_login/js' => 'application#js', as: :css

 <script src="<%= any_login.js_path  %>"></script>
 # ... same for CSS

  def js
     send_file "../somepath/../appllication.js", content_type: 'text/javascript'
  end

# same for css too
```

and no more changes :)

rule is -> less code = less bugs

Reply:
Just my 2 cents since I tested the first commit. There were a couple of issues with webpacker (only) and don't think that the webpacker approach works in theory.

Reply:
Most of the code for the Webpacker route was borrowed from RailsAdmin, I may have missed importing a few pieces that may have caused an issue.

Reply:
I've gone ahead and restructured this into this format, @OskarsEzerins apologies to keep bothering you with this but do you mind testing one more time on the new commit? It should also be compatible.

Reply:
Gotcha. I could look into it if it becomes necessary. Though @igorkasyanchuk suggestion seems to be a bit more straight forward.

Reply:
> I've gone ahead and restructured this into this format, @OskarsEzerins apologies to keep bothering you with this but do you mind testing one more time on the new commit? It should also be compatible.

It works great now. It isn't dependent on any asset host.

Reply:
Thank you for testing, glad to hear it works!